actions/run/src/run.ts

```diff
@@ -103,3 +115,24 @@ async function spawnCommand(command: string, args: string[]): Promise<string> {
     })
   })
 }
+
+function getTargetsFromEarthfile(target: string, earthfile: string): string[] {
```

Comment:
Perhaps I'm missing something here, but why are we parsing the targets in Typescript? Isn't this something we're already doing with the CLI?

Reply:
Yes this is what I concern about, it is quite redundant to `getTarget` in `earthfile.go`, but isn't the `scan` in `cli/cmd/main.go` only set the output for the path that contains earthfile?  Correct me if I'm wrong